.github/workflows/yml.yml

```diff
@@ -0,0 +1,18 @@
+name: YAML check
+
+on:
+  pull_request:
+    branches: [main]
+    paths:
+      - "**/*.yml"
+      - "**/*.yaml"
+      - ".github/workflows/yml.yml"
+      - ".github/yamllint.yml"
+
+jobs:
+  yaml-check:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Validate YAML file
+        run: yamllint -c .github/yamllint.yml .
```

Comment:
I think we should move `/.github/yamllint.yml` to `/.yamllint.yml` for proximity with `.golangci.yml`. It will also make it unnecessary to specify `-c` while making it the [default config](https://yamllint.readthedocs.io/en/stable/configuration.html#:~:text=a%20file%20named%20.yamllint%2C%20.yamllint.yaml%2C%20or%20.yamllint.yml%20in%20the%20current%20working%20directory) should someone run `yamllint` locally.